relay-metrics/src/protocol.rs

```diff
@@ -5,11 +5,10 @@ use std::iter::FusedIterator;
 use hash32::{FnvHasher, Hasher};
 use serde::{Deserialize, Serialize};
 
-pub use relay_common::UnixTimestamp;
-
 #[doc(inline)]
 pub use relay_common::{
     CustomUnit, DurationUnit, FractionUnit, InformationUnit, MetricUnit, ParseMetricUnitError,
+    UnixTimestamp,
```

Comment:
Was there a reason this was not under `#[doc(inline)]` before?

Reply:
I don't think so, looks like an oversight.